local/blockchain.go

```diff
@@ -50,10 +50,10 @@ var (
 )
 
 type blockchainInfo struct {
-	vmName       string
-	vmID         ids.ID
-	subnetID     ids.ID
-	blockchainID ids.ID
```

Comment:
This value is a bit ambiguous to me. To me, `chainID` is the evm parameter (like 43114 for c-chain) and `blockchainID` is the p-chain parameter. If we want the p-chain parameter, we should keep this as `blockchainID`